### ISSUE TITLE
What happened to FreeSports on 28.2°E?

### ISSUE THREAD

**Issue by BrianG61UK** [oe-alliance/AutoBouquetsMaker#228]
What happened to FreeSports on 28.2°E?
It used to be FTA and the Sky UK provider correctly put it as channel 422.
Now it seems to be FTV and, as far as I can see, is completely ignored by the Sky UK provider.
Does it need to be whitelisted or something?
The only way I can watch it is to do a scan of 11426V.

**Comment by AbuBaniaz** [comment]
There were a few posts on some forums about it becoming FreeToView, i.e requiring a card with entitlements. 

However, I have just run ABM without hacks for Sky UK, London HD. It was on 422. Maybe the carriage fees issue has been resolved?

**Comment by BrianG61UK** [comment]
@AbuBaniaz Ah, thanks. I did not know there was a dispute. I saw it had gone from my Sky all channels ABM bouquet and investigated and discovered it was FTV and then failed when using the method you showed me to add the FTV channel back into the bouquet. ( using these, but updated:
https://www.world-of-satellite.com/showthread.php?63107-Expired-Sky-Card-FTV&p=500335&viewfull=1#post500335
https://www.world-of-satellite.com/showthread.php?47016-ABM-Hacks-Examples&p=391794&viewfull=1#post391794 )

The sat_282_sky_uk_CustomMix.xml file from your post [391794](https://www.world-of-satellite.com/showthread.php?47016-ABM-Hacks-Examples&p=391794&viewfull=1#post391794) (2nd URL above) seems to produce inconsistent results on my Zgemma H7. (Once made Python 3 compatible).
Occasionally bits are missing from the channel names, e.g. one of the ITV regions might show as just ITV in the csv file, or, more often, the CHID (channel_id) value for some channels will be 0 rather than the correct value. But running ABM multiple times over a few minutes and comparing the outputted csv files and choosing the biggest one seems to get me one that seems to be 100% correct. Very strange.
